docs/HowToUseSmartFSDumpVisualizer.md

```diff
@@ -14,26 +20,37 @@ To run SDV, please download and install Java 8 (or the latest Java) to a target
 
 >**Note** To use TizenRT IDE together, Java 8 should be installed.
 
-SDV can be released for Windows, Linux (Ubuntu), and Mac OS.
+SDV can be released for Linux (Ubuntu), Windows (Some functions are constrained), and Mac OS (Possible but not yet).
```

Comment:
`Mac OS (Possible but not yet).`
I don't think this information is necessary. 

how about writing the supported os as of now?

Reply:
Good, modified.